docker-compose-workflows.yml

```diff
@@ -14,19 +14,22 @@ services:
       - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_HOST=temporal
       - WORKFLOW_TEMPORAL_CLUSTER_FRONTEND_PORT=7233
       - WORKFLOW_BACKEND_HOST=http://api:3000
+      - CODE_EXECUTOR_INGRESS_DOMAIN=http://code-executor:3004
     networks:
       - frontend-network
       - backend-network
       - db-connector-network
       - db-ssh-connector-network
       - workflows-network
+      - code-executor-network
```

Comment:
do we need to also add this to `workflow-worker` container? or do we never call code-executor directly from the worker

Reply:
wait i think this is done already